packages/@tinacms/forms/src/form.ts

```diff
@@ -269,13 +271,13 @@ function updateEverything<S>(form: FormApi<any>, values: S) {
 }
 
 function updateSelectively<S>(form: FormApi<any>, values: S, prefix?: string) {
-  const activePath: string = form.getState().active!
+  const activePath = form.getState().active!
```

Comment:
In this situation, `activePath` can be a `string | number | symbol`.  We were assuming `string`.  I kept the typing intact and changed the logic below as `startsWith()` is only available to `string`.